stdlib/ballerina-builtin/src/main/ballerina/runtime/Package.md

```diff
@@ -0,0 +1,85 @@
+## Package overview
+
+This package includes functions to interact with the runtime, the invocation context and to manage errors.
+
```

Comment:
Can you please add explanations/samples for other functions/record types in the runtime package also.

Reply:
Added generic descriptions about the errors and the functions in the runtime.bal.